load.go

```diff
@@ -56,17 +57,21 @@ func (c *loadCmd) loadApps() {
 		}
 
 		op := &appOperation{
-			action:    "UPDATE",
-			time:      time.Now(),
+			baseOperation: baseOperation{
```

Comment:
[nit] I'd create a `newBaseOperation(action string)` function that already sets the time internally, just to remove some noise from the load functions.